src/utilities/eventDescription.js

```diff
@@ -0,0 +1,2 @@
+export const APPROVED = 'Approved';
```

Comment:
might be worth putting this file in either a 'constants' folder or calling it 'eventDescriptionConstants.js' - probably the former

Reply:
ok, i made this change